openupgrade_scripts/scripts/website_event_track/16.0.1.3/pre-migration.py

```diff
@@ -0,0 +1,24 @@
+from openupgradelib import openupgrade
+
+
+def create_sequence(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        ALTER TABLE event_track_location
+        ADD COLUMN IF NOT EXISTS sequence integer
+        """,
+    )
+
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE event_track_location
+        SET sequence = 10
+        """,
+    )
```

Comment:
The new column should have already been created by the module and have the default value of `10`, could you please check if this is actually needed?

Reply:
Indeed it is that way.